clients/admin-ui/src/features/config-wizard/config-wizard.slice.ts

```diff
@@ -0,0 +1,88 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+import { HYDRATE } from "next-redux-wrapper";
+
+import type { AppState } from "~/app/store";
+
+import { REVIEW_STEPS, STEPS } from "./constants";
+
+export interface State {
+  step: number;
+  reviewStep: number;
+  systemFidesKey: string;
+}
+
+const initialState: State = {
+  step: 1,
+  reviewStep: 1,
+  systemFidesKey: "",
+};
+
+export const slice = createSlice({
+  name: "configWizard",
+  initialState,
+  reducers: {
+    /**
+     * With no argument: increment to the next step.
+     * With an argument: switch to that step.
+     */
+    changeStep: (draftState, action: PayloadAction<number | undefined>) => {
```

Comment:
This is a use of the "draft" prefix. RTK automatically [uses immer](https://redux-toolkit.js.org/usage/immer-reducers#mutating-and-returning-state) which can save a lot of redundant object spread/copy by giving you an object that is safe to modify and overall easier to work with.